meta-resin-common/recipes-support/resin-device-uuid/resin-device-uuid/resin-device-uuid

```diff
@@ -41,7 +41,7 @@ if [ -z "$CONFIG_PATH" ]; then
     exit 1
 fi
 
-if [ ! -f "$CONFIG_PATH" ]; then
+if [ -s "$CONFIG_PATH" ]; then
     echo "[INFO] resin-device-uuid: Config file is missing. Creating.."
     echo '{}' > $CONFIG_PATH
```

Comment:
I'm not sure why we have this here. I think it should bail out on missing config.json.